.gitignore

```diff
@@ -1,2 +1,4 @@
 bin/**
 vendor/**
+.idea
```

Comment:
This should go in your global gitignore and not in the project's

Reply:
true :: sorry